Index: PcClub_Api/src/main/resources/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>spring.application.name=PcClub\r\nspring.datasource.url=jdbc:postgresql://localhost:8080/PcClub\r\nspring.datasource.username=postgres\r\nspring.datasource.password=admin\r\nspring.jpa.generate-ddl=true\r\njwt.secret=hbfjhsdbf8734yr3bf34KNLQLM34dglbfi438f34b4y394f34fhbd9y3bdbBV36JOAB\r\njwt.expiration.string=1h
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/PcClub_Api/src/main/resources/application.properties b/PcClub_Api/src/main/resources/application.properties
--- a/PcClub_Api/src/main/resources/application.properties	(revision eb0f1f318b0e23252051c05e4f5dd8374577313b)
+++ b/PcClub_Api/src/main/resources/application.properties	(date 1734810324984)
@@ -1,5 +1,5 @@
 spring.application.name=PcClub
-spring.datasource.url=jdbc:postgresql://localhost:8080/PcClub
+spring.datasource.url=jdbc:postgresql://localhost:5432/PcClub
 spring.datasource.username=postgres
 spring.datasource.password=admin
 spring.jpa.generate-ddl=true
